packages/near-fast-auth-signer-e2e-tests/firebase.ts

```diff
@@ -0,0 +1,74 @@
+import admin from 'firebase-admin';
+
+import { serviceAccount } from './utils/serviceAccount';
+
+admin.initializeApp({
```

Comment:
I'm not sure, but I believe some of the functions for firebase are already create by @hcho112 . 

Like this, I think both do the same thing, no? https://github.com/near/fast-auth-signer/blob/b6bfb4516130c7de6798a5c3ec5b99d28b5d8fae/packages/near-fast-auth-signer-e2e-tests/utils/firebase.ts#L29

If that's the case, please re-use the ones from Danny and include yours on the same file.

Reply:
Both were actually created by @hcho112, this PR has nothing to do with both files, I only rebased after @hcho112's merge to be update.

Both files are already in `main`.

Reply:
@benmalcom if you look at latest main, you will notice that firebase.ts is moved under utils/firebase.ts

so this file is redundant. If you have added new functions, these should be allocated to utils/firebase.ts

Reply:
@hcho112 you're right, I see it's useless now, i've removed it.